kokoro/windows/continuous.bat

```diff
@@ -3,7 +3,9 @@ cd github\google-cloud-eclipse
 
 set CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
 call gcloud.cmd components update --quiet
+@echo on
 call gcloud.cmd components install app-engine-java --quiet
```

Comment:
Ah I didn't realize call allowed the batch/command to affect the callers environment.